cpp/libcudf_kafka/CMakeLists.txt

```diff
@@ -0,0 +1,169 @@
+#=============================================================================
+# Copyright (c) 2018-2020, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#=============================================================================
+cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
+
+project(CUDA_DATAFRAME VERSION 0.15.0 LANGUAGES C CXX CUDA)
+
+if(NOT CMAKE_CUDA_COMPILER)
+  message(SEND_ERROR "CMake cannot locate a CUDA compiler")
+endif(NOT CMAKE_CUDA_COMPILER)
```

Comment:
Future TODO: Remove CUDA language since this is just cpp code and use the `FindCUDAToolkit` cmake package instead.

Reply:
Do you want me to do this now? Since it says Future TODO I am unsure?

Reply:
Nope, just a note for future reference, maybe add a comment in the CMake that since we have no actual device code we don't need the CUDA language at all.

Reply:
roger